mt32emu/src/mt32emu.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/include/mt32emu
```

Comment:
BTW, there is also CMAKE_INSTALL_INCLUDEDIR variable, if we go GNUInstallDirs way.